File: gemma4-skills-os/skills/coding/setup_ruflo.ts
---
source_repo: https://github.com/ruvnet/ruflo
source_file: v3/__tests__/setup.ts
license: MIT
category: skills/coding
imported_at: 2026-04-19
---

/**
 * V3 Test Setup
 * Global test configuration for Vitest
 */

import { beforeAll, afterAll, vi } from 'vitest';

// Mock console.warn for cleaner test output
beforeAll(() => {
  vi.spyOn(console, 'warn').mockImplementation(() => {});
});

afterAll(() => {
  vi.restoreAllMocks();
});

// Increase timeout for integration tests
vi.setConfig({
  testTimeout: 30000,
});

File: gemma4-skills-os/skills/coding/setup_ruflo.py
---
source_repo: https://github.com/ruvnet/ruflo
source_file: v2/examples/flask-api-sparc/setup.py
license: MIT
category: skills/coding
imported_at: 2026-04-19
---

from setuptools import setup, find_packages

setup(
    name="rest-api",
    version="1.0.0",
    packages=find_packages(),
    install_requires=[
        'flask>=2.3.0',
        'flask-sqlalchemy>=3.0.0',
        'flask-cors>=4.0.0',
        'python-dotenv>=1.0.0',
    ],
    extras_require={
        'dev': [
            'pytest>=7.4.0',
            'pytest-cov>=4.1.0',
            'black>=23.7.0',
            'flake8>=6.0.0',
        ]
    }
)
